vendor.conf

```
@@ -31,29 +31,29 @@ github.com/Microsoft/go-winio                       v0.4.14
 github.com/Microsoft/hcsshim                        v0.8.9
 github.com/opencontainers/go-digest                 v1.0.0
 github.com/opencontainers/image-spec                v1.0.1
-github.com/opencontainers/runc                      v1.0.0-rc10
-github.com/opencontainers/runtime-spec              v1.0.2
+github.com/opencontainers/runc                      v1.0.0-rc91
+github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875
```

😞 looks like this is needed for https://github.com/opencontainers/runc/pull/2424. Let me open a request to have a new tag

Meanwhile, could you perhaps add a comment to describe which version this is?

```suggestion
github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875 # v1.0.2-8-g237cc4f
```